Readme.md

```diff
@@ -1,95 +1,97 @@
+[![Windows CI](https://ci.appveyor.com/api/projects/status/github/sass/libsass?svg=true)](https://ci.appveyor.com/project/sass/libsass/branch/master "Appveyor CI")
+[![Coverage Status](https://img.shields.io/coveralls/sass/libsass.svg)](https://coveralls.io/r/sass/libsass?branch=feature%2Ftest-travis-ci-3 "Code coverage of spec tests")
+[![Percentage of issues still open](http://isitmaintained.com/badge/open/sass/libsass.svg)](http://isitmaintained.com/project/sass/libsass "Percentage of issues still open")
+[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/sass/libsass.svg)](http://isitmaintained.com/project/sass/libsass "Average time to resolve an issue")
+[![Bountysource](https://www.bountysource.com/badge/tracker?tracker_id=283068)](https://www.bountysource.com/trackers/283068-libsass?utm_source=283068&utm_medium=shield&utm_campaign=TRACKER_BADGE "Bountysource")
+[![Join us](https://libsass-slack.herokuapp.com/badge.svg)](https://libsass-slack.herokuapp.com/ "Slack communication channels")
+
+
+[LibSass](https://github.com/sass/libsass "LibSass GitHub Project") is just a library!
+If you want to use LibSass to compile Sass, you need an implementer. Some
+implementations are only bindings into other programming languages. But most also
+ship with a command line interface (CLI) you can use directly. There is also
+[SassC](https://github.com/sass/sassc), which is the official lightweight
+CLI tool built by the same people as LibSass.
+
+### Excerpt of "sanctioned" implementations:
+
+- https://github.com/sass/node-sass (Node.js)
+- https://github.com/sass/perl-libsass (Perl)
+- https://github.com/wellington/go-libsass (Go)
+- https://github.com/sass/sassc-ruby (Ruby)
+- https://github.com/medialize/sass.js (JS)
+
+This list does not say anything about the quality of either the listed or not listed [implementations](docs/implementations.md)!  
+The authors of the listed projects above are just known to work regularly together with LibSass developers.
+ 
 About
 -----
 
-LibSass is a C/C++ port of the Sass CSS precompiler. The original version was written in Ruby, but this version is meant for efficiency and portability.
-
-This library strives to be light, simple, and easy to build and integrate with a variety of platforms and languages.
+LibSass is a C++ port of the original Ruby Sass CSS compiler with a [C API](docs/api-doc.md).
+We coded LibSass with portability and efficiency in mind. You can expect LibSass to be magnitudes
+faster than Ruby Sass. We would even go as far and claim that it is the fastest CSS compiler
+around, which should be the case given it's written in C++.
 
 Developing
 ----------
 
-As you may have noticed, the LibSass repo itself has
-no executables and no tests. Oh noes! How can you develop???
+As noted above, the LibSass repository does not contain any binaries or other way to execute
```

Comment:
Not sure what you mean exactly? IMO it adds a bit more information than the old wording. And I don't think the old wording was any less confusing? IMO the whole paragraph does not make much sense, but I've left it in since it already was there.